src/Tribe/Service_Providers/Context.php

```diff
@@ -264,6 +273,11 @@ static function ( $post_type ) {
 				],
 				'event_post_type'      => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( TEC::POSTTYPE, false ) );
```

Comment:
Return `Tribe__Context::NOT_FOUND` here, not `false` unless you really mean `false`.
If this sub-location does not return `Tribe__Context::NOT_FOUND`, then the lookup of the location will stop.